crates/typst-library/src/model/outline.rs

```diff
@@ -79,8 +163,10 @@ pub struct OutlineElem {
 
     /// The type of element to include in the outline.
     ///
-    /// To list figures containing a specific kind of element, like a table, you
-    /// can write `{figure.where(kind: table)}`.
+    /// To list figures containing a specific kind of element, like an image or
+    /// a table, you can specify the desired kind in a where selector. See the
```

Comment:
```suggestion
    /// a table, you can specify the desired kind in a [`where`]($function.where) selector. See the
```

Linking to the corresponding page here would probably be beneficial.

Regardless of whether a link is added here, "where" should probably be typeset as inline code.